packages/react/src/components/DataTable/Table.tsx

```diff
@@ -66,13 +74,69 @@ export const Table = ({
     [`${prefix}--data-table--sticky-header`]: stickyHeader,
     [`${prefix}--data-table--visible-overflow-menu`]: !overflowMenuOnHover,
   });
+
+  const setTableAlignment = useCallback(() => {
+    const fragment = document.createDocumentFragment();
+    const canvas = document.createElement('canvas');
+    fragment.appendChild(canvas);
+    const context = canvas.getContext('2d');
```

Comment:
Does jsdom support canvas? Can we write tests against this functionality?

Reply:
I think it should with the added canvas dependency, the tests that were failing are passing since adding that